api/src/routesStudents.js

```diff
@@ -93,43 +90,24 @@ async function remove(req, res, next) {
   }
 }
 
-async function update(req, res, next) {
```

Comment:
this was gross and i'll fight anyone who tries to defend it to me

Reply:
i concur